pytx/pytx/vocabulary.py

```diff
@@ -220,6 +225,19 @@ class PagingCursor(object):
     AFTER = 'after'
 
 
+class Reaction(object):
+
+    """
+    Vocabulary for describing a reaction.
+    """
+
+    HELPFUL = 'HELPFUL'
+    NOT_HELPFUL = 'NOT_HELPFUL'
+    WANT_MORE_INFO = 'WANT_MORE_INFO'
+    SAW_THIS_TOO = 'SAW_THIS_TOO'
+    OUTDATED = 'OUTDATED'
+
+
```

Comment:
Greetings @hammem! Was that 'nit' like nitpicking?
Greetings @mgoffin! Just alphabetize this one in descending order (fan favorite) 👍